package/etc/conf.d/conflib/raw/app-symantec_dlp.conf

```diff
@@ -0,0 +1,18 @@
+block parser symantec_dlp-parser() {    
+ channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index('netauth')
+                sourcetype('symantec:dlp:syslog')
+                vendor_product("symantec_dlp")
+            );              
+        };          
+
+   };
+};
+application symantec_dlp[sc4s-syslog] {
+	filter { 
+        message('incident_id' type(string) flags(substring));
```

Comment:
I can imagine the string "incident_id" could appear in other syslog source this isn't safe enough for a parser we need something more unique or the filter will need to be host/ip based

Reply:
@rfaircloth-splunk Updated as per the review comments